packages/cli/src/dirCommand.ts

```diff
@@ -100,7 +100,9 @@ export const dirCommand: SvgrCommand = async (
       const prettierRcConfig = opts.runtimeConfig
         ? await resolveConfig(filepath, { editorconfig: true })
         : {}
+      const prettierFileInfo = await getFileInfo(filepath)
```

Comment:
Need to be in a `Promise.all`

Reply:
Did some more digging in the prettier source code. `getFileInfo` ends up using resolveConfig again and inferring based on filepath. So to avoid extra work (and extra IO) it's better to just pass in file path as an option. Updated the code and verified by running the CLI.